ci/cloudbuild/builds/cmake-install.sh

```diff
@@ -149,6 +149,7 @@ done < <(find "${INSTALL_PREFIX}" -type f -print0)
 
 for repo_root in "ci/verify_current_targets" "ci/verify_deprecated_targets"; do
   out_dir="cmake-out/$(basename "${repo_root}")-out"
+  rm -f "${out_dir}/CMakeCache.txt"
```

Comment:
This is so that we can run the `cmake-install` build multiple times on a machine where this file is persistent (e.g., our `--docker` builds). The problem is that this file caches the _temporary_ install location from the previous invocation.

Reply:
For a separate PR: maybe the install location should not be temporary?  `/h/test-install` or `cmake-out/test-install`?

Reply:
I think we want a fully clean install each time. Otherwise, if we _removed_ an artifact in one build, the artifact may still exist on disk and we may not notice that we're still depending on the removed artifact.... well, our CI builds in GCB would notice, but we wouldn't notice on our local machines until we blew away the install directory and rebuilt. Anyway, that's why I originally installed in a new dir every time.